src/components/addProductWishModal/AddProductWishModal.test.tsx

```diff
@@ -0,0 +1,37 @@
+import 'react-native';
+import React from 'react';
+import {jest, test, describe, expect} from '@jest/globals';
+import AddProductWishModal from './AddProductWishModal';
+import {Image} from 'react-native';
+import {SUCCESS_PNG_PATH} from '../../utils/images';
+import {render} from '@testing-library/react-native';
+
+const mockedNavigate = jest.fn();
+
+jest.mock('@react-navigation/native', () => {
+  const actualNav = jest.requireActual('@react-navigation/native');
+  return {
+    actualNav,
+    useNavigation: () => ({
+      navigate: mockedNavigate,
+    }),
+  };
+});
+
+describe('AddProductWishModal tests', () => {
+  test('Renders the AddProductWishModal correctly', () => {
+    const tree = render(<AddProductWishModal />).toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+
+  test('AddProductWishModal render works correctly', () => {
+    render(<AddProductWishModal />);
+  });
+
+  test('Component AddProductWishModal contains image', () => {
```

Comment:
It'd be more verbose to specify which kind of image it contains, like 'Success image' or some other.
If there're multiple variants - it'd be the best to cover all of them separately.

Reply:
Fixed. Good note. On the other hand, if I reused a modal window in which there could be different pictures. In this case, the picture is only 1.